### PR TITLE
New Quote section

### PR DESCRIPTION
**Pull request by andrealbiac**
Created a new section with a quote (wip) of Austin about the BuidlGuidl, to add a bit more context at the start of the site, cause the hero Title felt a bit too short.

This is how it looks:
Captura de pantalla 2024-09-28 a las 17 06 43
Captura de pantalla 2024-09-28 a las 17 06 51

This one was very straightforward so nothing to report. It was a great exercise to re-remember some flexbox and other tags that I didn't know. 

But same, let me know if something is off or it should have a different structure/ anything. 
Thanks! 🤗

### CODE REVIEW

File: packages/nextjs/pages/index.tsx [hunk 1]
```diff
@@ -48,6 +48,31 @@ const Home: NextPage<{
...
+              alt="austin griffith"
+              width={120}
+              height={120}
+              className="rounded-[150px] shadow-xl"
+            />
+          </div>
+          {/* Quote Text */}
+          <div className="md:max-w-[80%] lg:max-w-[70%] flex flex-col items-center lg:items-start">
+            <p className="font-thin text-2xl lg:text-2xl text-center lg:text-left mb-10 lg:mb-4">
+              “The BuidlGuidl is a curated group of developers and designers with the goal of on boarding new developers
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:67]: "on boarding" is ok? Or should be "onboarding"? I usually see "onboarding" even tho the corrector doesn't seem to like it.

File: packages/nextjs/pages/index.tsx [hunk 2]
```diff
@@ -48,6 +48,31 @@ const Home: NextPage<{
...
+              src="/assets/austin-pfp.png"
+              alt="austin griffith"
+              width={120}
+              height={120}
+              className="rounded-[150px] shadow-xl"
+            />
+          </div>
+          {/* Quote Text */}
+          <div className="md:max-w-[80%] lg:max-w-[70%] flex flex-col items-center lg:items-start">
+            <p className="font-thin text-2xl lg:text-2xl text-center lg:text-left mb-10 lg:mb-4">
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:66]: Maybe we could use text-xl for smaller devices? Don't know if it was intended to set `text-2xl lg:text-2xl` or it's a typo and wanted to set `text-xl lg:text-2xl`

**Review by Pabl0cks — COMMENTED** [review]
Looks good to me! Nice job @andrealbiac 

Only added a couple of nitpicks

File: packages/nextjs/pages/index.tsx (hunk 1 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:67]: Yeah true, I also think it should be onboarding. Good call!

File: packages/nextjs/pages/index.tsx (hunk 2 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:66]: Yeah I agree, I think I changed it to `text-2xl` cause it looked a bit small but with the computer screen is not very realistic anyway. Changing it back to `text-xl`

**Comment by andrealbiac** [comment]
Thanks @Pabl0cks ! Just commited a fix with the tiny adjustments 🙌🏻

**Review by Pabl0cks — APPROVED** [review]
Thanks for the fast fixes!

**Pull request merged**